src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs

```diff
@@ -165,6 +165,11 @@ public override void Remove(View child)
             if (child == null || child.GetParent() == null) // Early out if child null.
                 return;
 
+            if (child.Parent != this)
```

Comment:
please add exception noti in document comment. =>
<exception cref=xxx> xxx </exception>
(example) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/Window.cs#L69

Reply:
done